config/opensearch_dashboards.yml

```diff
@@ -164,4 +164,8 @@
 
 # Set the value of this setting to true to capture region blocked warnings and errors 
 # for your map rendering services.
-# map.showRegionBlockedWarning: false
\ No newline at end of file
+# map.showRegionBlockedWarning: false
+
+# Set the value of this setting to false to suppress search usage telemetry
+# for reducing the load of OpenSearch cluster.
+data.searchUsageTelemetry.enabled: false
```

Comment:
Can we comment this line since it is already default to false similar to `showRegionBlockedWarning` above? All other parameters are using its default values without explicit assignment.

Reply:
Elaborate a bit more: the system recognizes the parameter `data.searchUsageTelemetry.enabled` in yml through the schema (and the default value) defined in `src/plugins/data/config.ts`. Thus `     enabled: schema.boolean({ defaultValue: false }),` is already sufficient. It is safe to mark this line as commented and keep here for just instruction purpose.

Reply:
I agree lets make sure this is default commented on and rely on the schema definition defaulting this to false.

Reply:
Thanks, I updated the opensearch_dashboards.yml